src/NuGet.Core/NuGet.Build.Tasks/GetCentralPackageVersionsTask.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+
+namespace NuGet.Build.Tasks
+{
+    public class GetCentralPackageVersionsTask : Task
+    {
+        /// <summary>
+        /// Full path to the msbuild project.
+        /// </summary>
+        [Required]
+        public string ProjectUniqueName { get; set; }
+
+        /// <summary>
+        /// Central package versions.
+        /// </summary>
+        [Required]
+        public ITaskItem[] CentralPackageVersions { get; set; }
+
+        /// <summary>
+        /// Target frameworks to apply this for. If empty this applies to all.
+        /// </summary>
+        public string TargetFrameworks { get; set; }
+
+        /// <summary>
+        /// Output items
+        /// </summary>
+        [Output]
+        public ITaskItem[] RestoreGraphItems { get; set; }
+
+        public override bool Execute()
+        {
+
+#if DEBUG
+            System.Diagnostics.Debugger.Launch();
+#endif
```

Comment:
In the past, `Debugger.Launch` wasn't available in .NET Core. That might have changed, but please use the same [pattern as `RestoreTask`](https://github.com/NuGet/NuGet.Client/blob/e67a08d76583040e1aab01a79a509a8be088d1c6/src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs#L77-L94). I won't want to debug every time I run the binary compiled in debug mode, so I don't want to wait for the debugger to launch just to run the task.

Reply:
Removed the code path.